webapp/src/utils/index.tsx

```diff
@@ -3,23 +3,21 @@
  */
 
 import React from 'react';
-import Cookies from 'js-cookie';
 
 import {Button} from '@brightscout/mattermost-ui-library';
 
 import Constants, {SubscriptionType, RecordType, KnowledgeRecordDataLabelConfigKey, RecordDataLabelConfigKey, CONNECT_ACCOUNT_LINK} from 'src/plugin_constants';
 
 import {id as pluginId} from '../manifest';
 
-const getBaseUrls = (): {
+const getBaseUrls = (mmSiteUrl?: string): {
```

Comment:
Easiest way to default to empty string. If we don't default to empty string, `undefined` will show up in the outout

```suggestion
const getBaseUrls = (mmSiteUrl = ''): {
```